internal/handlers/handlers.go

```diff
@@ -1 +1,69 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	http.ServeFile(w, r, "../Sprint_6_final/index.html")
```

Comment:
Запускаем из корня проекта, так что пути тоже относительно корня
```suggestion
	http.ServeFile(w, r, "./index.html")
```